gum/backend-x86/gumstalker-x86.c

```diff
@@ -3032,6 +3032,10 @@ gum_exec_block_virtualize_branch_insn (GumExecBlock * block,
     gum_x86_relocator_skip_one_no_label (gc->relocator);
     gum_exec_block_write_call_invoke_code (block, &target, gc);
   }
+  else if (insn->ci->id == X86_INS_JMP) {
+    if (ctx->stalker->any_probes_attached)
+      gum_exec_block_write_call_probe_code (block, &target, gc);
```

Comment:
We should move this block down to inside the final `else` [here](https://github.com/frida/frida-gum/blob/24bf115b777c30d8383e7d046ec867e0390d8b25/gum/backend-x86/gumstalker-x86.c#L3081) to avoid breaking instrumentation of `JMP` instructions.